### PR TITLE
feat(static): add LINK token, update USDC.e and USDT.e token names to USDC and USDT

### PR DESCRIPTION
**Pull request by evalenti-simtlix**
update USDC.e and USDT.e token names and fix test tokens in static and address to support Chainlink

STLX-11940

### CODE REVIEW

File: modules/statics/src/coins.ts [hunk 1]
```diff
@@ -1209,17 +1198,24 @@ export const coins = CoinMap.fromCoins([
...
+    'Test Chainlink',
+    18,
+    '0x0b9d5d9136855f6fec3c0993fee6e9ce8a297846',
+    UnderlyingAsset['avaxc:LINK']
+  ),
+  tavaxErc20(
+    'tavaxc:USDT',
     'Test Tether USD',
     6,
     '0xc7198437980c041c805a1edcba50c1ce5db95118',
```
Comment (jicli) [inline modules/statics/src/coins.ts:1211]: should this be updated to '0x9702230a8ea53601f5cd2dc00fdbc13d4df4a8c7'?

**Review by jrafael-simtlix — APPROVED** [review]
LGFM

File: modules/statics/src/coins.ts [hunk 2]
```diff
@@ -1209,17 +1198,24 @@ export const coins = CoinMap.fromCoins([
...
-    UnderlyingAsset['avaxc:USDT.E']
+    UnderlyingAsset['avaxc:USDT']
   ),
   tavaxErc20(
-    'tavaxc:USDC.E',
+    'tavaxc:USDC',
     'Test USD Coin',
     6,
     '0xa7d7079b0fead91f3e65f86e8915cb59c1a4c664',
```
Comment (jicli) [inline modules/statics/src/coins.ts:1218]: as well as this should be updated

File: modules/statics/src/coins.ts (hunk 1 above)
Comment (jrafael-simtlix) [reply modules/statics/src/coins.ts:1211]: @shiksb could you confirm USDT and USDC testnet contract address?

Comment (shiksb) [reply modules/statics/src/coins.ts]: look ok

**Review by shiksb — APPROVED** [review]

**Review by tylerlevine — APPROVED** [review]
nice work

**Pull request merged**